src/main/java/me/moodcat/backend/rooms/RoomInstance.java

```diff
@@ -182,6 +188,37 @@ protected void playNext(final Song song) {
 
         hasChanged.set(true);
     }
+    
+    private void processVotes(final Song previousSong) {
+        AtomicInteger nettoVotes = new AtomicInteger();
+        
+        this.votes.values().stream().forEach((vote) -> {
```

Comment:
The strategy is indeed another solution. The first approach wouldn't work since we want to acquire the netto votes, not the positive bruto.